task_DEV-3/EntryPoint.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Text;
+
+namespace task_DEV_3
+{
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            if (!(args.Length == 2))
+            {
+                Console.WriteLine("Error! Incorrect format of the data!");
+                return;
+            }
+
+            long inputNumber;
+            int baseOfNewSystem;
+

```

Comment:
Rework to Parse and handle exceptions (as agreed in chat) instead of  check returned bools